static/katex.less

```diff
@@ -399,23 +399,22 @@
     }
 
     .sqrt {
-        > .sqrt-sign {
-            position: relative;
-        }
-
-        .sqrt-line {
-            display: inline-block;
-            position: relative;
-            width: 100%;
-            border-bottom-style: solid;
-        }
-
         > .root {
             // These values are taken from the definition of `\r@@t`,
             // `\mkern 5mu` and `\mkern -10mu`.
             margin-left: 5*@mu;
             margin-right: -10*@mu;
         }
+
+        > span > span > .vlist {
+            text-align: left;
+        }
```

Comment:
In a numerator or a denominator, `text-align` is set to center. When I place a `\sqrt` into either one, I have to protect the surd from that alignment or else I get this: 
![align](https://user-images.githubusercontent.com/16403058/29345935-f59fad56-81f6-11e7-80a3-e293765296d1.PNG)